lua/lualine/themes/base16.lua

```diff
@@ -90,6 +90,34 @@ local function setup_base16_nvim()
   }
 end
 
+local function setup_tinted_nvim()
+  -- Continue to load tinted-nvim
+  local loaded, tinted = pcall(require, 'tinted-colorscheme')
+
+  if not loaded then
+    add_notice(
```

Comment:
This is problematic as this will show this notice even when base16_nvim is present